content/apps/introduction.md

```diff
@@ -0,0 +1,21 @@
+# What's an App ?
+
+::: warning
+The Apps are only available in the **6.0 version** and higher and for **SaaS** customers.
+:::
+
+Before you start developing an App, it's important to understand what Apps can do and how they interact with the PIM.
+
+An App is a service that can be easily connected to a PIM, directly from the PIM, 
+with a streamlined process of activation and configuration.
+
+An App can extend the PIM functionalities through the REST API, like a Connector, 
+by retrieving or updating data and integrate it with external services.
+
+The only constraint is that an App must have a single activation URL and should be hosted by its creator.
+
+In this documentation, we will guide you through the steps of building an App.
+
+::: info
+For more information on Apps usage, please read our help center article [some article](https://help.akeneo.com).
```

Comment:
```suggestion
For more information on Apps usage, please read our help center articles [How to connect Akeneo PIM with third parties](https://help.akeneo.com/pim/serenity/articles/how-to-connect-my-pim-with-apps.html) and [Manage your Apps](https://help.akeneo.com/pim/serenity/articles/manage-your-apps.html).
```

Reply:
These articles will be available on our Help center as soon as we release the feature.